index.html

```diff
@@ -192,11 +216,27 @@ <h1 id="hire-me-h1">I am available for freelance work.</h1>
         </div>
     </div>
     <!-- subscribe -->
-    <div class="container">
-        <div class="row">
-            SUBSCRIBE
+    <div class="row subscribe-section">
+        <div class="subscribe-section">
+            <h2><strong>Subscribe</strong> to newsletter</h2>
+            <div class="flex-parent snowflake">
+                <div class="flex-child-edge"></div>
+                <div class="flex-child-text">&#42;</div>
+                <div class="flex-child-edge"></div>
+               
+              </div>
+            <p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at this layout.</p>
+        </div>
+        <div class="subscribe-form">
+            <form id ="subscribeform" action="#" NoValidate>
+                <input class="subscribe-input" id="subscribe-email" type="email" placeholder="Email">
+                <div class="subscribe-error"></div>
+                <div class="subscribe-errorIcon">!</div>
+                <button class="subscribe-submit" id="subscribe-click" type="submit"> Subscribe</button>
```

Comment:
trūksta submit'ui hover'io, kur invertuojasi spalvos. 

ir truputį atrodo proporcijos ne visai tokios, subscribe eilutė visa ilgoka, placeholder'is didokas - ne esminiai dalykai tikrai : )

Reply:
Hoveris uždėtas ( be animacijos, bet ar jos reikia?) eilutė patrumpinau, placeholderį sumažinau, dėl proporcijų turbut reikia visiems apsitart kokio dydžio bus h2, -*-, p ir pagal tai lygiuoti